Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: The rarely used datatype of long double is not supported by SWIG
----

--- a/src/saga_core/saga_api/mat_tools.h
+++ b/src/saga_core/saga_api/mat_tools.h
@@ -117,7 +117,7 @@
 #define N_MEGABYTE_BYTES			0x100000
 
 //---------------------------------------------------------
-#define M_ALMOST_ZERO				0.001l
+#define M_ALMOST_ZERO				0.001
 #define M_TINY						(1.0e-20)
 
 //---------------------------------------------------------
